src/jdbc/cubrid/jdbc/net/BrokerHandler.java

```diff
@@ -62,11 +62,17 @@ public static Socket connectBroker(String ip, int port, boolean useSSL, int time
           if (code < 0) {
           // in here, all errors are sent by only a broker
           // the error greater than -10000 is sent by old broker
-            if (code > -10000) {
+          if (code > -10000) {
               code -= 9000;
-            }
-            throw new UJciException(code);
-          } else if (code == 0) {
+          }
+		
```

Comment:
Please leave a comment here that there is a legacy error that prints the error code.